satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -426,60 +426,61 @@ delete serial_number (
 // for preventing duplicate serial numbers
 create used_serial ()
 
-// --- bucket roll up tables --- //
+// --- bucket tally tables --- //
 
-model bucket_bandwidth_rollup (
+model bucket_bandwidth_tally (
```

Comment:
This isn't a tally, it's a rollup.

Reply:
could you please explain why you think it is a rollup and not a tally?

We decided on this change because of the functionality of the existing tally and rollup services.  The existing tally service counts up the occurrences at some interval and then the rollup service rolls up those counts into 1 day rollups.

Now with all the recent changes, the SA is incrementing the bucket_bandwidth_rollup/tally records when it receives the orders from the SN. That is replacing the functionality of the existing tally service. So that is why we changed the name from rollup to tally.